e2e/app/page/data-page.ts

```diff
@@ -162,8 +170,11 @@ export default class DataPage extends AuthenticatedPage {
    * @param {string} conceptsetName
    */
   async deleteConceptSet(conceptsetName: string): Promise<string> {
-    const datasetCard = await DataResourceCard.findCard(this.page, conceptsetName);
-    const menu = datasetCard.getEllipsis();
+    const card = await DataResourceCard.findCard(this.page, conceptsetName);
```

Comment:
nit: can we call this something like `conceptSetCard` to be consistent with how the other cards are named?

Reply:
Done. Thank you.